platform/android/java/lib/res/mipmap-anydpi-v26/icon.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!--
+WARNING: The contents of this file must always match the constant defined in export_plugin.cpp.
```

Comment:
Can you include the full path to `export_plugin.cpp`, there are multiple of it, one for each platform.

See https://github.com/godotengine/godot/blob/d741a646a5ee20e77c1d0d9d4fbfed03d7b8dcc3/platform/android/java/app/config.gradle#L4 for reference.

Reply:
@m4gr3d I’ve applied the changes. Please take another look.